src/puppetlabs/ring_middleware/core.clj

```diff
@@ -209,7 +295,7 @@
                                                :message msg}})
                         :plain (plain-response code msg))))]
      (fn [request]
-       (try (handler request)
+       (sling/try+ (handler request)
```

Comment:
True.

This is definitely unnecessary, when pulling in the additional handlers they used a slingshot `try+`/`throw+` pattern. I simply moved everything existing to use that pattern where possible, as well as slingshot by default.  Which looking back over it using slingshot regardless of the try/catch needs is a rather silly, and completely unnecessary, normalization.

Particularly, in light of the fact that I couldn't get the [schema errors middleware](https://github.com/puppetlabs/ring-middleware/pull/28/files#diff-01da1d044c8bc8e6c87bddbbf3793a02R270) to use slingshot.